packages/devtools_app/lib/src/flutter/app.dart

```diff
@@ -151,10 +172,12 @@ class DevToolsAppState extends State<DevToolsApp> {
   }
 }
 
-/// A [WidgetBuilder] that takes an additional map of URL query parameters.
+/// A [WidgetBuilder] that takes an additional map of URL query parameters and
+/// args.
 typedef UrlParametersBuilder = Widget Function(
   BuildContext,
   Map<String, String>,
+  Object args,
```

Comment:
make args a named parameter and give it type SnapshotArguments if that is the only case we are using it?

Reply:
done